src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -375,6 +389,63 @@ The zero register is extended with zero metadata and a cleared valid tag: this i
 
 NOTE: Instruction encodings refer to *c* registers when they use `cs1`, `cd`, etc. instead of `rs1`, `rd` (which reads/writes the XLEN-bit subset of the register).
 
+==== Added CSRs
+
+{cheri_base32_ext_name} and {cheri_base64_ext_name} add the capability CSR shown in
+xref:utidc-csrnames-added[xrefstyle=short].
+
+[[utidc-csrnames-added]]
+.Unprivileged capability CSRs added in {cheri_base32_ext_name} and {cheri_base64_ext_name}
+[%autowidth,float="center",align="center",cols="<,<,<,<",options="header"]
+|===
+|CLEN CSR|Address|Permissions|Description
+|<<utidc>>|0x480|RW, <<asr_perm>> required for writes, not reads|User Default Data Capability
+|===
+
+[#utidc,reftext="utidc"]
+===== User Thread Identifier Capability (utidc)
+
+The <<utidc>> register is used to identify the current software thread in user mode.
+On reset the valid tag of <<utidc>> will be set to 0 and the remainder of the data is UNSPECIFIED.
```

Comment:
We can't talk about reset in unpriv part of spec. And we also can't mandate what the priv execution environment sets this to, so best to drop it.
```suggestion
```

Reply:
I do mention system init in unpriv as we have to talk about a source of infinite-cap.
But I think you're right - maybe the reset value guidance should be in priv (annoyingly)